docs/developer_setup.md

```diff
@@ -170,6 +170,12 @@ git ls-files | xargs sed -i 's/A\.B\.C/X\.Y\.Z/g'
 
 Manually inspect the changes to filter out false positives.
 
+---
+## Access Acoustic STAGE
+
+- Login via https://sso.mozilla.com/acoustic
+- Select *Pod 9*
```

Comment:
This also requires getting being granted access by our Acoustic admins. Don't know if that's important to mention here.